### PR TITLE
Uncoupling UK and EU course fees

### PR DESCRIPTION
**Pull request by jesseyuen**
This entry gives some context to recent updates about uncoupling UK and EU course fees post Brexit.

### CODE REVIEW

File: app/posts/publish-teacher-training-courses/2020-10-01-uncoupling-UK-and-EU-course-fees.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+---
+title: Uncoupling UK and EU course fees
+description: Allowing providers to specify different course fees for UK and EU/international students
+date: 2020-10-01
```
Comment (edwardhorsford) [inline app/posts/publish-teacher-training-courses/2020-10-01-uncoupling-UK-and-EU-course-fees.md:4]: Needs updating?

**Review by edwardhorsford — APPROVED** [review]
Looks good to me! one query about date.

File: app/posts/publish-teacher-training-courses/2020-10-01-uncoupling-UK-and-EU-course-fees.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+{% from "figure/macro.njk" import appFigure with context %}
+{{ appFigure({
+  image: {
+    path: "/images/publish-teacher-training-courses/uncoupling-UK-and-EU-course-fees/",
+    file: "original-edit-course-fees.png",
+    alt: "Screenshot the original edit course length and fees page."
+  }
+}) }}
+
+## Design Notes
```
Comment (paulrobertlloyd) [inline app/posts/publish-teacher-training-courses/2020-10-01-uncoupling-UK-and-EU-course-fees.md]:
```suggestion
## Design notes
```

File: app/posts/publish-teacher-training-courses/2020-10-01-uncoupling-UK-and-EU-course-fees.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+This update uncouples EU student fees from UK fees, leaving the optional international field for fees for students from any non-UK location.
+
+## Further questions
+
+If there are no international fees displayed, is the course still open to international students? The Find course pages currently do not address this scenario and this is an additional piece of work that will need to follow.
```
Comment (paulrobertlloyd) [inline app/posts/publish-teacher-training-courses/2020-10-01-uncoupling-UK-and-EU-course-fees.md:38]: 👀

File: app/posts/publish-teacher-training-courses/2020-10-01-uncoupling-UK-and-EU-course-fees.md (hunk 1 above)
Comment (jesseyuen) [reply app/posts/publish-teacher-training-courses/2020-10-01-uncoupling-UK-and-EU-course-fees.md:4]: I date my entries to refelect an estimate of when the work was done. 👍

**Pull request merged**